ospec/ospec.js

```diff
@@ -236,6 +237,13 @@ module.exports = new function init(name) {
 
 	function report() {
 		var status = 0
+
+		if (typeof reporter === "function") {
```

Comment:
This should still be a single line to match the rest of the codebase.

Reply:
Got it in. Thanks for your patience.